05.wc/wc.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  paths, target_options = options_and_paths
+  text_metadata_collection =
+    if paths.empty?
+      [create_text_metadata(input: $stdin.read)]
+    else
+      build_text_metadata(paths)
+    end
+  width = calculate_output_width(text_metadata_collection, target_options)
+  text_metadata_collection.each do |text_metadata|
+    render(target_options, text_metadata, width)
```

Comment:
```suggestion
    render(text_metadata, target_options, width)
```

`text_metadata`をrenderするのが主目的なので、`text_metadata`が最初の引数にある方がいいですね。
オプションの類いは引数の順番としては後ろの方に来ることが多いです。

Reply:
[返り値や引数の順番を修正](https://github.com/s-tone-gs/ruby-practices/pull/10/commits/d31e9b17ab8640e180ff8b68e5f9bab1cc4925be)にて修正しました！